resources/quality/weedo_x40/0.4_generic_PLA_Extra_Fine.inst.cfg

```diff
@@ -0,0 +1,38 @@
+[general]
+version = 4
+name = Default - Extra Fine
+definition = weedo_x40
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = extra fine
+weight = 1
+material = generic_pla_175
+variant = 0.4mm Nozzle
+
+
+[values]
+layer_height = 0.12
+layer_height_0 = =round(layer_height * 1.5, 2)
+support_fan_enable = True
+support_supported_skin_fan_speed = 100
+cool_fan_full_at_height = =layer_height_0 + 2 * layer_height
+cool_fan_speed_max = =cool_fan_speed
+cool_min_speed = 7
+infill_line_width = =round(line_width * 0.42 / 0.35, 2)
+machine_nozzle_cool_down_speed = 0.75
+machine_nozzle_heat_up_speed = 1.6
+material_standby_temperature = 100
+prime_tower_enable = False
+skin_overlap = 10
+
+speed_print = 30
+speed_layer_0 = =round(speed_print / 2.5, 2)
+speed_print_layer_0 = =speed_layer_0
+speed_infill = =speed_print
+speed_support_interface = =speed_print
+speed_wall = =round(speed_print / 2, 2)
+speed_wall_0 = =speed_wall
+speed_wall_x = =speed_wall
+speed_topbottom = =speed_wall
```

Comment:
Why set this for every profile? If you want this to be default for your printer, just change it in the definition. This makes it a lot easier to see what the profile actually changed.

I'd advise to double check all the settings in the profiles and check if you really need to set them.